service/header/rpc.go

```diff
@@ -0,0 +1,114 @@
+package header
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+
+	"github.com/celestiaorg/celestia-node/node/rpc"
+)
+
+const (
+	headerByHeightEndpoint = "/header"
+	dahByHeightEndpoint    = "/dah"
```

Comment:
Why do we need a separate /dah request if the header already contains dah? Is this an optimization that we should care rn?

Reply:
In Optimint situation looks like:
 - I need data from given namespace at given height
 - I need DAH to get data by namespace
 - I don't care about rest of the header

Getting only DAH is an optimization, but getting namespaced data by block height would be even better (reducing number of requests from 2 to 1) ;)

Reply:
note: After reading comments in another PR I understand why 2 requests are required (at least now).